website/src/pages/Home.tsx

```diff
@@ -0,0 +1,178 @@
+      <a href="https://fr.wikipedia.org/wiki/R%C3%A9volution_num%C3%A9rique">révolution</a>, qui a radicalement
+      bouleversé notre façon de communiquer <em>et de nous informer</em>. En contrepartie, un nombre croissant de
+      problématiques liées à l'information émergent, notamment dans la diffusion de celle-ci par les médias. Les
+      articles relatant des faits hors du commun étant plus attrayants, les contenus sont parfois plus
+      sensationnalistes que vrais.
+    </p>
+
+    <p>
+      Mais avons-nous <strong>les bons outils</strong> pour réfléchir ensemble, intelligemment, face à cette
+      abondance d'information sur internet ? Réagir à l'information a pour ambition d'apporter des solutions à ces
+      problèmes, en proposant une plateforme qui <em>vous</em> permet de réagir librement aux médias sur
+      internet, comme des articles de presse ou des vidéos sur YouTube.
+    </p>
+
+    <DownloadExtension>Installer l'extension chrome</DownloadExtension>
+
+    <Title id="Comment">Comment ?</Title>
+
+    <div className="what">
+      <Image
+        maximize
+        src={imageYoutubRi}
+        containerClassName="what-image-flex-container"
+        alt="screenshot youtube réagir à l'information"
+      />
+      <div className="what-item-wrapper">
+        { sentences.map(({ text, subtext }, n) => (
+          <div key={n} className="what-item">
+            <h3>{ text }</h3>
+            <p>{ subtext }</p>
+          </div>
+        )) }
+      </div>
+    </div>
+
+    <p>
+      Nous tentons de rassembler sur cette plateforme, des personnes bienveillantes, attentives aux biais, qui savent
+      écouter et partager leurs opinions, tout en gardant un œil critique face aux arguments qu'on leur présente.
+      Si vous voulez mieux comprendre l'information et participer à des échanges constructifs sur internet, prenez une
+      seconde pour ajouter un marque page, et pourquoi pas <Link href="/usage#inscription">vous inscrire</Link> !
+    </p>
+
+    <p className="wording">
+      Mais il est certes difficile de constituer une telle communauté. Un point central du projet repose sur{' '}
+      <Link href="/charte">la charte</Link>, qui tente d'apporter un cadre propice aux débats. Consacrez une dizaine de
+      minutes à sa lecture, avant de vous inscrire. Et si vous souhaitez apporter une évolution des règles, n'hésitez
+      pas à <Link href="/faq#contact">envoyer un message</Link> à l'équipe qui développe le projet.
+    </p>
+
+    <p>
+      Vous voulez en savoir plus ? La page <Link href="/motivations">motivations</Link> explique plus en détail les
+      raisons pour lesquelles le projet a vu le jour, et son état d'esprit. Et pour commencer à utiliser l'extension dès
+      maintenant, rendez-vous sur la page <Link href="/utilisation">utilisation</Link>. A bientôt sur internet !
+    </p>
+
+    <div className="break" />
```

Comment:
add some padding to the page, as it is non-sense to declare an html tag "just to make space"... that's what margins and paddings are for